.github/workflows/ci-actions.yml

```diff
@@ -362,6 +362,8 @@ jobs:
             postgres: "true"
             timeout: 45
             test-modules: >
+              elasticsearch-lowlevel
+              elasticsearch-highlevel
```

Comment:
This would need some consistency with the extension names.

Reply:
Fixed